app/scripts/modules/core/src/pipeline/config/triggers/runAsUserSelector.component.ts

```diff
@@ -1,25 +1,38 @@
-import { module } from 'angular';
+import { IComponentOptions, IController, module } from 'angular';
+import { SETTINGS } from 'core/config/settings';
 
-class RunAsUserSelectorComponent implements ng.IComponentOptions {
+class RunAsUserSelectorController implements IController {
+  public enabled = true;
+
+  public $onInit(): void {
+    this.enabled = SETTINGS.feature['fiatEnabled'] && !SETTINGS.feature['managedServiceAccounts'];
```

Comment:
You don't need the bracket notation - can just get the key `SETTINGS.feature.managedServiceAccounts`. It's possible that the TS compiler will complain, but in that case you should add `managedServiceaccounts` to `settings.ts`, which types `settings.js`.